app/assets/stylesheets/pages/_machines.scss

```diff
@@ -0,0 +1,270 @@
+.devices {
+  min-height: 60vh;
+
+  b {
+    color: $c-pink;
+    font-weight: 600;
+  }
+}
+
+.device__selection {
+  img {
+    align-self: center;
+    height: auto;
+    max-width: 135px;
+    min-width: 52px;
+    width: 20%;
+  }
+
+  ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  li {
+    background-color: #f5f5f5;
+    cursor: pointer;
+    display: flex;
+    flex-basis: 33.333%;
+    justify-content: center;
+    text-align: center;
+    transition: all 0.3s ease-in-out;
+
+    &:hover,
+    &:focus {
+      background-color: lighten($c-pink, 5%);
+      text-decoration: underline;
+    }
+  }
+
+  li:not(.device-selected) {
+    opacity: 0.5;
+  }
+
+  a {
+    align-self: flex-end;
+    color: $c-text;
+    display: flex;
+    flex-basis: 100%;
+    flex-direction: column;
+    padding: rem(23) rem(0);
+    text-decoration: none;
+
+    @include breakpoint(min-width 1440px) {
+      flex-direction: row;
+      justify-content: center;
+
+      h2 {
+        margin-left: 4%;
+        margin-bottom: 3vh;
+        align-self: center;
+      }
```

Comment:
I think you can get rid of the `margin-bottom` and `align-self` styles here by just adding `align-items: center` to the `a`